lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def initialize
+    reset!
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts ""
+  end
+
+  def position(input)
+    cells[input.to_i - 1]
+  end
+
+  def full?
+    full_board = @cells.all? do |space|
```

Comment:
we could turn this into a small one liner, since Ruby implicitly returns the last line of code in a method.
```
@cells.all? { |space| space == "X" || space == "O" }
```